### DISCUSSION TITLE
Update username and password

### DISCUSSION THREAD

**Discussion by ThePeePs** [xannor/ha_reolink_rest#36] [Ideas]
Can a way be added to be able to update/change the username and password for each camera without having to ether manually edit the `core.config_entries` file, or remove and re-add the camera?

**Comment by xannor** [comment]
You should be able to view the integration and it should report a login error, then you should get a prompt to provide a new password.

**Comment by ThePeePs** [comment]
ah, ok. I was looking to change it right after I had updated the login on the camera, I didn't wait for the prompt.
